packages/commons/src/AxeTestEnvironment.cs

```diff
@@ -1,11 +1,33 @@
 namespace Deque.AxeCore.Commons
 {
+    /// <summary>
+    /// Information about the current browser or node application that ran the audit.
+    /// </summary>
     public class AxeTestEnvironment
     {
+        /// <summary>
+        /// User Agent
+        /// </summary>
         public string UserAgent { get; set; }
+
+        /// <summary>
+        /// Window Width.
+        /// </summary>
         public int WindowWidth { get; set; }
+
+        /// <summary>
+        /// Window Height.
+        /// </summary>
         public int WindowHeight { get; set; }
+
+        /// <summary>
+        /// Orientation Type.
+        /// </summary>
         public string OrientationType { get; set; }
+
+        /// <summary>
+        /// Orientation Angle.
```

Comment:
It would be nice for this to clarify whether it's in degrees or radians

Reply:
From https://w3c.github.io/screen-orientation/#dom-screenorientation-angle, looks like it's degrees. Will update the comment accordingly.